package.json

```diff
@@ -26,7 +26,8 @@
     "flow-typed": "flow-typed",
     "build:tokens": "lerna run tokens --scope bpk-tokens",
     "build:svgs": "lerna run svgs --scope bpk-svgs",
-    "build": "npm run build:tokens && npm run build:svgs && lerna run build",
+    "build:css": "node scripts/build-sass-in-place.js",
```

Comment:
We have `https://github.com/Skyscanner/backpack-node-sass` for this :)

Reply:
Oh nice - I'll use that!